internal/pkg/cli/pipeline_init.go

```diff
@@ -554,14 +551,8 @@ func (o *initPipelineOpts) secretName() string {
 	return fmt.Sprintf(fmtSecretName, o.appName, o.repoName)
 }
 
-func (o *initPipelineOpts) pipelineName() (string, error) {
-	if o.provider == ghProviderName {
-		return fmt.Sprintf(fmtGHPipelineName, o.appName, o.githubOwner, o.repoName), nil
-	}
-	if o.provider == ccProviderName {
-		return fmt.Sprintf(fmtCCPipelineName, o.appName, o.repoName), nil
-	}
-	return "", fmt.Errorf("unable to create pipeline name for repo %s from provider %s", o.repoName, o.provider)
+func (o *initPipelineOpts) pipelineName() string {
+	return fmt.Sprintf(fmtPipelineName, o.appName, o.repoName)
```

Comment:
Should we generate a name by default that is always <= 100 chars? I think right now it's still possible that our generated names are super long.

We should not do this but I was having fun: https://play.golang.org/p/oJvOr5z5e7M . I think let's do the simple thing and truncate at 100 chars

Reply:
Ha, that's great! 
Added truncation.